### PR TITLE
Add check to add listener hosts and test

### PR DESCRIPTION
**Pull request by laura-ding**
close https://jira.nebula-graph.io/browse/ONCALL-109

depends on https://github.com/vesoft-inc/nebula-common/pull/470

### CODE REVIEW

File: tests/admin/test_listener.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+import time
+
+from tests.common.nebula_test_suite import NebulaTestSuite
+
+class TestListener(NebulaTestSuite):
+    @classmethod
+    def prepare(self):
```
Comment (yixinglu) [inline tests/admin/test_listener.py:14]: need not to implement an empty function

File: tests/admin/test_listener.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        resp = self.client.execute('REMOVE LISTENER ELASTICSEARCH;')
+        self.check_resp_succeeded(resp)
+
+        # CHECK listener
+        resp = self.client.execute('SHOW LISTENER;')
+        self.check_resp_succeeded(resp)
+        self.check_result(resp, [])
+
+    @classmethod
+    def cleanup(self):
```
Comment (yixinglu) [inline tests/admin/test_listener.py:49]: ditto

**Review by yixinglu — APPROVED** [review]

File: src/validator/AdminValidator.cpp [hunk 3]
```diff
@@ -235,9 +235,26 @@ Status ShowSnapshotsValidator::toPlan() {
 
 Status AddListenerValidator::validateImpl() {
     auto sentence = static_cast<AddListenerSentence*>(sentence_);
-    if (sentence->listeners()->hosts().empty()) {
+    auto hosts = sentence->listeners()->hosts();
+    if (hosts.empty()) {
         return Status::SemanticError("Listener hosts should not be empty");
     }
+
+    // check the hosts, if the hosts the same with storage, return error
```
Comment (Shylock-Hg) [inline src/validator/AdminValidator.cpp:243]: I think you could check is the listener host exist.

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Pull request merged**